akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEvent.scala

```diff
@@ -21,6 +21,12 @@ import scala.annotation.tailrec
 
 object ServerSentEvent {
 
+  sealed trait EventIdValue
+  case object ResetId extends EventIdValue
+  final case class Id(id: String) extends EventIdValue {
```

Comment:
There is a special value - the empty string - with which you can tell the client that it should _not_ reconnect with a `Last-Event-Id`-Header . I thought it might be nicer not to expose this detail as a raw empty string in the api.